docs/tokens.md

```diff
@@ -0,0 +1,63 @@
+# Tokens && Fees
+
+## Supported tokens
+
+RIF Rollup can be used to transfer RBTC and ERC20 tokens. The full list of currently supported tokens is available under
+this [link]() (opens new window).
+
+> DISCLAIMER: Listing of a token on RIF Rollup does not represent an endorsement for this token by the RIF Rollup team,
+> IOVLabs, or any other entity. We do not have competence and authority to analyze the token business model and smart
+> contract security guarantees of a particular token. Please use any token on your own risk and judgement.
+
+## Listing a token
+
+### as user
+
+To list a new token on zkSync, go the [tokens page] (opens new window)of the explorer and click the "Add New Token"
+button and follow the instructions.
```

Comment:
There's no "Add New Token" in our version of the explorer.

Reply:
is it not? how come?

Reply:
removed